cmd/root.go

```diff
@@ -112,18 +112,18 @@ func executePhases(cmd cobra.Command, progress *phases.Progress) error {
 		progress.PrivateKey = &cmdFlagPrivateKey
 	}
 	if progress.PrivateKey == nil {
-		key, err := phases.PrivateKey()
+		_, privKey, _, err := phases.PrivateKey()
 		if err != nil {
 			return err
 		}
-		progress.PrivateKey = &key
+		progress.PrivateKey = &privKey
 	}
 
 	if cmd.Flags().Changed(cmdFlagKeyBitriseYML) {
 		progress.BitriseYML = &cmdFlagBitriseYML
 	}
 	if progress.BitriseYML == nil {
-		yml, err := phases.BitriseYML()
+		yml, _, err := phases.BitriseYML()
```

Comment:
the first workflow should also be part of the progress, as we will need it to start the first build